src/views/Referrals/Referrals.js

```diff
@@ -0,0 +1,601 @@
+import React, { useEffect, useState } from "react";
+import { useWeb3React } from "@web3-react/core";
+
+import Card from "../../components/Common/Card";
+import SEO from "../../components/Common/SEO";
+import Tab from "../../components/Tab/Tab";
+import Footer from "../../Footer";
+import {
+  useChainId,
+  getPageTitle,
+  formatAmount,
+  USD_DECIMALS,
+  helperToast,
+  formatDate,
+  getTokenInfo,
+  getExplorerUrl,
+  shortenAddress,
+  ARBITRUM,
+  bigNumberify,
+  REFERRAL_CODE_QUERY_PARAMS,
+  isHashZero,
+} from "../../Helpers";
+import { decodeReferralCode, useReferralsData } from "../../Api/referrals";
+
+import "./Referrals.css";
+import { registerReferralCode, setTraderReferralCodeByUser, useInfoTokens, useUserReferralCode } from "../../Api";
+import { utils } from "ethers";
+import { BiCopy, BiEditAlt } from "react-icons/bi";
+import Tooltip from "../../components/Tooltip/Tooltip";
+import { useCopyToClipboard } from "react-use";
+import Loader from "../../components/Common/Loader";
+import Modal from "../../components/Modal/Modal";
+import { RiQuestionLine } from "react-icons/ri";
+import { FiPlus } from "react-icons/fi";
+
+const getSampleReferrarStat = (code) => {
+  return {
+    discountUsd: bigNumberify(0),
+    referralCode: code,
+    totalRebateUsd: bigNumberify(0),
+    tradedReferralsCount: 0,
+    trades: 0,
+    volume: bigNumberify(0),
+  };
+};
+
+const TRADERS = "Traders";
+const AFFILIATES = "Affiliates";
+let TAB_OPTIONS = [TRADERS, AFFILIATES];
+
+function getDollarValue(value) {
```

Comment:
i guess we use "usd" everywhere, may be better to use it here as well for consistency